test/components/number.test.js

```diff
@@ -58,157 +69,166 @@ describe('<Number />', () => {
 
     it('should default to valueDecimals when no ticks are supplied', () => {
       const decimals = 3;
-      const component = shallow(<Number value={1.4444} valueDecimals={decimals} />);
-      expect(component.find('FormattedNumber').prop('minimumFractionDigits')).to.equal(decimals);
+      const component = shallow(<Number.WrappedComponent intl={intl} value={1.4444} valueDecimals={decimals} />);
+      expect(component.find('.number > span').text()).to.equal('1.444');
```

Comment:
If you think it is cleaner we could send in valueClass="some-class" and then use `expect(component.find('.some-class').text()).to.equal('1.444');` in the following tests